src/components/__tests__/CurrentRefinements.js

```diff
@@ -177,11 +177,8 @@ it('calls the Panel mixin with `canRefine`', async () => {
 
   expect(mapStateToCanRefine()).toBe(true);
 
-  await wrapper.setData({
-    state: {
-      items: [],
-    },
-  });
+  wrapper.vm.state.items = [];
```

Comment:
https://github.com/vuejs/vue-test-utils-next/issues/766

this will be reverted once we get the new VTU release.